src/services/metadata.ts

```diff
@@ -0,0 +1,34 @@
+import { ServiceSceneCustomState, ServiceSceneState } from '../Components/ServiceScene/ServiceScene';
+
+let metadataService: MetadataService;
+
+export function initializeMetadataService(): void {
+  if (!metadataService) {
+    metadataService = new MetadataService();
+  }
+}
```

Comment:
This should be enough and then you don't need any initialization, right?

```suggestion
const metadataService = new MetadataService();
```

Reply:
Yeah this does seem to work, but in app-o11y and grafana core we always init singletons like this? 
Scares me a bit...

Reply:
I guess it comes down to how the bundles are built? Using a scoped variable within the module works as long as the file is included once across all bundles? I'll ask someone smarter then myself

Reply:
It's definitely not a big deal.

Reply:
Great question though, but let's merge and I'll follow up